browser-test/browser-test-compose.yml

```diff
@@ -17,7 +17,7 @@ services:
     profiles: ["azure"]
     container_name: azurite-test
     ports:
-    - 10000
+      - 10000:10000
```

Comment:
previously this just mapped the port in the docker compose network but we need it mapped to the bridged (localhost) network as well

Reply:
Optional: the port:port syntax isn't intuitive to me, but I don't have yml readability either :D Consider if an inline comment would help.

Reply:
Here's doc for that: https://docs.docker.com/compose/compose-file/compose-file-v3/#ports

I'm hesitant to leave a comment since there is a lot in this file that isn't particularly intuitive without reading the docs. Also the "ports" config option is used in a bunch of places and commenting each of them seems cumbersome.